src/Compilers/Core/Portable/FileSystem/PathUtilities.cs

```diff
@@ -444,5 +460,182 @@ internal static bool ContainsPathComponent(string path, string component, bool i
 
             return false;
         }
+
+        /// <summary>
+        /// Gets a path relative to a directory.
+        /// </summary>
+        public static string GetRelativePath(string directory, string fullPath)
```

Comment:
How exactly does this differ from the CoreCLR implementation: https://github.com/dotnet/corefx/blob/7bfa0c77da5a14bdef41e7530fee41d5fb3fd912/src/System.Runtime.Extensions/src/System/IO/Path.cs#L483 ?